CMakeLists.txt

```diff
@@ -483,6 +488,8 @@ get_target_property(flatbuffers_dir ${flatbuffers_target}
 get_filename_component(flatbuffers_dir ${flatbuffers_dir} DIRECTORY) # <prefix>
 
 find_package(yaml-cpp 0.6.2 REQUIRED HINTS ${YAML_CPP_ROOT})
+get_filename_component(yaml-cpp_dir "${YAML_CPP_INCLUDE_DIR}" ABSOLUTE)
```

Comment:
Can you name this `yaml_cpp_dir` instead?

Also, wouldn't it make more sense to use `get_target_property` to get the include directory?

Reply:
`YAML_CPP_INCLUDE_DIR` is already provided by `yaml-cpp-config.cmake`, why should we ignore that?

Reply:
Can you confirm that works with both 0.6.2 and 0.6.3? Because somewhere in between they reworked their CMake completely w.r.t to packaging. If this solution is portable then yes, keep it.

Reply:
We used the same variable before this change. The config file has only one revision: https://github.com/jbeder/yaml-cpp/commits/master/yaml-cpp-config.cmake.in